db/migrate/20200404085605_create_activities.rb

```diff
@@ -1,8 +1,8 @@
 class CreateActivities < ActiveRecord::Migration[6.0]
   def change
     create_table :activities do |t|
-      t.string :label
-      t.text :description
+      t.string :label, null: false
```

Comment:
aha...hmm..i would not modify existing migrations. that could lead to problems when releasing. because the db means, it has already be done. simply add a new one to do the change. but this is in discussion anyways. e.g. my main comment.